md/bdm-access-control.md

```diff
@@ -314,3 +315,157 @@ Deploy your access control file.
 Connect onto the portal as a user with the profile *Order picker*. Go to your application: the customer data and the order price are always empty.
 Connect now as a user with the profile *Sales*. On your application, the customer data are displayed except the address. The price of the product is now available.
+
+
+### III - Instances protection
+
+Instance protection is not supported as a part of the BDM Access Control feature. However it is still possible to solve most use cases using the [rest-api authorizations](rest-api-authorization.md).
+Following is a step by step guide on how to realize one such (simple) use case:
+
+::: info
+The method shown in the example below is not a new 7.7 feature, it is present in the Community version since 7.0 ( although the location of the files and the push/pull procedure changed in 7.6.0).
+It allows to protect the bdm instances by limiting the access to the BDM query requests that retrieve the object instances rather than to the instances themselves.
+:::
+
+**Scenario:**
+
+We will look into a mark contestation process for a school. Students can make requests about their marks to their teachers through a BPM process.
+Each teacher teaches a different subject. A teacher can only access requests made concerning their subject.
+
+The step by step guide assumes a working Bonita Studio v. 7.7.x +, with the Acme organization deployed.
+
+**Step by step implementation:**
+
+**1. Define the BDM**
+
+We will use a relatively simple BDM for this use case: 
+
+1. an object Student with attributes:
+
+| Name | Type |
+|---|---|
+| fullname | STRING |
+
+2. an object Request with attributes:
+
+ | Name | Type |
+|---|---|
+| subject | STRING |
+| medicalComment | STRING |
+| content | STRING |
+| student | Student |
+
+Subject and Student are mandatory fields.
+We also define a custom query on the Request object, *findBySubject*:
+
+```
+SELECT r
+FROM Request r
+WHERE r.subject= :subject
+ORDER BY r.persistenceId
+
+```
+
+**2. Define the profiles**
+
+We will need a few new profiles, mapped to one or more users. We need one profile per subject, as well as a **Teacher** profile (for all the teachers). A maths teacher would therefore have **MathematicsTeacher** and a **Teacher** profile. In the rest of the guide we will use 3 profiles **MathematicsTeacher**, **PhysicsTeacher** and **Teacher**, mapped to **helen.kelly**, **jan.fischer** and both respectively.
+For the details on how to easily create and map those profiles see the [Profile Creation](profileCreation.md) page.
+
+**3. Create a process and generate some request instances**
+
+You will need some instances of the request object, as well as some students. To create them, follow the steps discribed in the section  [II - Invoice](#bdmFilling).
+
```

Comment:
Tu user must create data with Mathematics and Physics as subjects, else the application won't work if he follows the next step, so it should be specified here .